contracts/crowdsale/Crowdsale.sol

```diff
@@ -10,7 +10,9 @@ import "../math/SafeMath.sol";
  * Crowdsales have a start and end timestamps, where investors can make
  * token purchases and the crowdsale will assign them tokens based
  * on a token per ETH rate. Funds collected are forwarded to a wallet
- * as they arrive.
+ * as they arrive. The contract requires a MintableToken that will be
+ * minted as contributions arrive, note that the crowdsale contract
+ * must be owner of the token in order to be able to mint it.
```

Comment:
We could enforce this in the `Crowdsale` constructor as:
```
require(_token.owner() == this);
```

On the other hand, we have had a lot of requests to be able to use the crowdsale with a non-mintable token, for example in a model where the crowdsale can _transfer_ pre-minted tokens. #554 proposed to solve this problem by having a "`Mintable`" interface with a possible implementation where `mint` doesn't create tokens but transfers from a pre-minted pool. In this case the crowdsale doesn't need to be the token owner.

Just throwing this out there because we might have to keep it in mind in the near future, to move forward towards something like #554 (which I personally want us to do).

Reply:
Hmm agree to *not* enforce it, to acommodate for non-ownable tokens in the future as you say.

Reply:
Are we comfortable with the owner being able to mint tokens arbitrarily before the crowdsale starts?

Reply:
I'd say yes, since it can be verified by anyone by looking at the total supply right before the sale starts. Besides, it's perfectly possible to do that now, simply by extending the crowdsale contract and spawning a mintable token that allows more than one owner to mint.